src/cmd/commit.ts

```diff
@@ -32,52 +42,82 @@ export const gitPush = async (branch: string): Promise<boolean> => {
     d.info(e.stdout)
     d.error(e.stderr)
     return false
-  } finally {
-    cd(rootDir)
   }
 }
 
-export const commit = async (): Promise<void> => {
-  const d = terminal('commit')
-  await validateValues()
-  d.info('Preparing values')
-  const values = await hfValues()
-  cd(env.ENV_DIR)
-  await preCommit()
-  await encrypt()
-  d.info('Committing values')
-  cd(env.ENV_DIR)
+const setDeplymentStatus = async (): Promise<void> => {
+  const status = await getOtomiDeploymentStatus()
+  if (status !== 'deployed') {
+    await nothrow($`kubectl -n ${otomiStatusNamespace} create cm ${otomiStatusCmName} --from-literal=status='deployed'`)
+    // Since status is an indicator of successful deployment, the generated passwords must be deleted later.
+    await nothrow($`kubectl delete secret ${otomiPasswordsSecretName}`)
```

Comment:
```suggestion
    await nothrow($`kubectl -n ${DEPLOYMENT_PASSWORDS_SECRET_NAMESPACE} create cm ${DEPLOYMENT_STATUS_CONFIGMAP_NAME} --from-literal=status='deployed'`)
    // Since status is an indicator of successful deployment, the generated passwords must be deleted later.
    await nothrow($`kubectl delete secret ${DEPLOYMENT_PASSWORDS_SECRET_NAME}`)
```

Reply:
not gonna change it